docs/content/preview/yugabyte-voyager/migrate/live-fall-back.md

```diff
@@ -717,11 +717,109 @@ target:
+
+```sh
+yb-voyager assess-migration --source-db-type postgresql \
+    --source-db-host hostname --source-db-user ybvoyager \
+    --source-db-password password --source-db-name dbname \
+    --source-db-schema schema1,schema2 --export-dir /path/to/export/dir
+```
+
+If you are using a [configuration file](../../reference/configuration-file/), use the following:
+
+```sh
+yb-voyager assess-migration --config-file <path-to-config-file>
+```
+
+  {{% /tab %}}
+
+  {{% tab header="Without source database connectivity" %}}
+
+PostgreSQL only. In situations where direct access to the source database is restricted, there is an alternative approach. Voyager includes packages with scripts for PostgreSQL at `/etc/yb-voyager/gather-assessment-metadata`.
+
+You can perform the following steps with these scripts:
+
+1. On a machine which has access to the source database, copy the scripts and install dependencies psql and pg_dump version 14 or later. Alternatively, you can install yb-voyager on the machine to automatically get the dependencies.
+
+1. Run the `yb-voyager-pg-gather-assessment-metadata.sh` script by providing the source connection string, the schema names, path to a directory where metadata will be saved, and an optional argument of an interval to capture the IOPS metadata of the source (in seconds with a default value of 120). For example:
+
+    ```sh
+    /path/to/yb-voyager-pg-gather-assessment-metadata.sh 'postgresql://ybvoyager@host:port/dbname' 'schema1|schema2' '/path/to/assessment_metadata_dir' '60'
+    ```
+
+1. Copy the metadata directory to the client machine on which voyager is installed, and run the `assess-migration` command by specifying the path to the metadata directory as follows:
+
+    ```sh
+    yb-voyager assess-migration --source-db-type postgresql \
+        --assessment-metadata-dir /path/to/assessment_metadata_dir --export-dir /path/to/export/dir
+    ```
+
+    If you are using a [configuration file](../../reference/configuration-file/), use the following:
+
+    ```sh
+    yb-voyager assess-migration --config-file <path-to-config-file>
+    ```
+
+    {{% /tab %}}
+
+    {{< /tabpane >}}
+
+1. The output is a migration assessment report, and its path is printed on the console. To view the assessment report, navigate to the **Migrations** tab in the [yugabyted UI](#configure-yugabyted-ui) at <http://127.0.0.1:15433> to see the available migrations.
+
+    {{< warning title="Important" >}}
+For the most accurate migration assessment, the source database must be actively handling its typical workloads at the time the metadata is gathered. This ensures that the recommendations for sharding strategies and cluster sizing are well-aligned with the database's real-world performance and operational needs.
+    {{< /warning >}}
+
+1. Resize your target YugabyteDB cluster in [Enhanced PostgreSQL Compatibility Mode](../../../develop/postgresql-compatibility/), based on the sizing recommendations in the assessment report.
```

Comment:
"Resize your target YugabyteDB cluster in Enhanced PostgreSQL Compatibility Mode" Doesn't quite make sense. They can resize the target; and they can enable EPCM; these are 2 separate things.
And Shouldn't they have already created the target db in EPCM?